src/material/button/_button-theme.scss

```diff
@@ -168,6 +168,10 @@
   @include mdc-helpers.using-mdc-typography($theme) {
     @include mdc-button.without-ripple($query: mdc-helpers.$mdc-typography-styles-query);
   }
+
+  .mat-mdc-raised-button {
+    line-height: inherit;
```

Comment:
Have we tried putting this in the component base styles? I'm fine with having it here if it helps us land the changes, but it technically should be in the base.

Reply:
When I tried putting these in the components base styles, I also ran into an issue where some teams just aren't pulling in theme styles. As you can imagine, setting the `line-height: inherit` at a _slightly_ higher specificity there resulted in a bunch of failures, so I moved the override over to the theme styles. This way, the override is right after the `line-height: 36px` coming from the `without-ripple` mixin on line 169, and it doesn't get pulled in for teams that aren't pulling in theme styles

Also, this will only be here temporarily. I need to land this change for each of the variants so that ripping out the `without-ripple` is feasible. Once these have landed, I'll then swap the `without-ripple` for our tokens and then I can remove these styles altogether

Reply:
Makes sense. It's not a blocker.